libcodechecker/server/client_db_access_handler.py

```diff
@@ -1731,56 +1731,41 @@ def massStoreRun(self, name, tag, version, b64zip, force):
         # Storing file contents from plist.
         file_path_to_id = {}
 
-        _, _, report_files = next(os.walk(report_dir), ([], [], []))
-        for f in report_files:
-            if not f.endswith('.plist'):
-                continue
-
-            LOG.debug("Parsing input file '" + f + "'")
-
-            try:
-                files, _ = plist_parser.parse_plist(
-                    os.path.join(report_dir, f))
-            except Exception as ex:
-                LOG.error('Parsing the plist failed: ' + str(ex))
+        for file_name, file_hash in filename2hash.iteritems():
+            source_file_name = os.path.join(source_root,
+                                            file_name.strip("/"))
+            source_file_name = os.path.realpath(source_file_name)
+            LOG.debug("Storing source file: " + source_file_name)
+
+            if not os.path.isfile(source_file_name):
+                # The file was not in the ZIP file, because we already have the
+                # content. Let's check if we already have a file record in the
+                # database or we need to add one.
+
+                LOG.debug(file_name + ' not found or already stored.')
+                fid = store_handler.addFileRecord(self.__Session(),
```

Comment:
Will this call diagnose if the hash is not yet in the database? (To catch mistakes in CodeChecker logic.)

Reply:
Yes